docs/dev/investigations/bluetooth/00-overview.md

```diff
@@ -1,73 +1,38 @@
-# Bluetooth Control Investigation
+# Bluetooth Control Investigations
 
-**Device:** Kobo Libra Colour (MTK Bluetooth chipset)
+This section documents the technical investigation and implementation details for Bluetooth control on different Kobo device types.
 
-## Overview
+## Device Support
 
-This investigation documents how to control Bluetooth on Kobo e-readers from the system level,
-enabling programmatic control without Nickel's UI. The investigation focuses on understanding the
-D-Bus interface, service initialization, and safe shutdown procedures.
+### MTK Devices
+- Uses MTK-specific Bluetooth implementation
+- D-Bus service: `com.kobo.mtk.bluedroid`
+- Custom command set and initialization sequence
+- See [MTK Documentation](./mtk/00-overview.md)
 
-## Key Findings
+### Non-MTK Devices (Libra 2, etc.)
+- Uses standard Linux BlueZ stack  
+- D-Bus service: `org.bluez`
+- Standard Bluetooth operations
+- See [Libra 2 Documentation](./libra-2/00-overview.md)
 
-### Custom D-Bus Wrapper
+## Architecture Overview
 
-Kobo does **NOT** expose the standard `org.bluez` D-Bus service. Instead, all BlueZ operations are
-routed through `com.kobo.mtk.bluedroid`.
+The plugin automatically detects the device type and uses the appropriate implementation:
 
-**Evidence:**
+1. **Device Detection**: Checks for MTK vs standard Bluetooth hardware
+2. **Service Discovery**: Uses appropriate D-Bus service (`com.kobo.mtk.bluedroid` vs `org.bluez`)  
+3. **Command Adaptation**: Executes device-specific command sequences
+4. **Input Handling**: Manages HID input devices consistently across platforms
```

Comment:
Since this is pure technical investigations, there is no need to explain how the plugin uses this in this section. 

Once this is actually implemented, it can be made part of of the dev docs, instead of the investigation docs.